Makefile

```diff
@@ -22,13 +17,5 @@ download-cosmos-proto:
 	mkdir -p proto/gogoproto
 	curl https://raw.githubusercontent.com/cosmos/cosmos-sdk/${cosmos-version}/third_party/proto/gogoproto/gogo.proto -o proto/gogoproto/gogo.proto
 
-gen-protoc-ts: 
-	mkdir -p ./client/src/types/generated/
-	ls proto/checkers | xargs -I {} protoc \
-		--plugin="./scripts/node_modules/.bin/protoc-gen-ts_proto" \
-		--ts_proto_out="./client/src/types/generated" \
-		--proto_path="./proto" \
-		--ts_proto_opt="esModuleInterop=true,forceLong=long,useOptionals=messages" \
-		checkers/{}
-	
-install-and-gen-protoc-ts: download-cosmos-proto install-protoc-gen-ts gen-protoc-ts
+gen-proto-ts: download-cosmos-proto install-gen-tool
```

Comment:
I would not install as part of `gen-proto-ts`. I am thinking of a situation where we run the `make` target from within Docker that has a global install of `telescope`.

Reply:
Yes, that's a good catch! npx can run from local or global. So I'll remove install on this!